site/src/atomics/ratio.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Ratio
```

Comment:
```suggestion
title: Aspect ratio
```
to align a little more with the property name?

Reply:
Went ahead and updated file names to match also.